backend/uclapi/resources/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^pcavailability$', views.get_pc_availability),
```

Comment:
Maybe this should be `desktop_availability`? I think using underscores to separate words would work well (it's quite Pythonic too and means our users don't need to use uppercase characters. Plus, all our parameters are named with underscores). @wilhelmklopp, do you think this would work?